udata/templates/dataset/resource/list-item.html

```diff
@@ -41,7 +41,7 @@ <h4 class="list-group-item-heading ellipsis">
     {{ resource.owner|avatar(52, 'resource-owner') }}
     {% endif %}
 
-    {% if can_edit_resource(resource, dataset) %}
+    {% if can_edit_resource(resource if resource.from_community else dataset) %}
```

Comment:
I prefer `if can_edit_resource(dataset) and can_edit_community_resource(resource)`, does it make sense?